pkg/controller/pd_control.go

```diff
@@ -328,13 +324,6 @@ func (pc *pdClient) DeleteStore(storeID uint64) error {
 		return err
 	}
 
-	// FIXME: We should not rely on error text
-	// Remove a tombstone store should returns "store has been removed"
-	bodyStr := string(body)
-	if strings.Contains(bodyStr, "store has been removed") {
-		return nil
-	}
-
```

Comment:
PD now actually returns a structured json response with an error code and an error code as a header.

Reply:
Yes, but the older version don't have the error code and header.

Return the error is ok for this situation. The `DeleteStore` function is reentrant.